addons/telemetry-coverage-bug1487578/install.rdf

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RDF xmlns="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:em="http://www.mozilla.org/2004/em-rdf#">
+  <Description about="urn:mozilla:install-manifest">
+    <em:id>telemetry-coverage-bug1487578@mozilla.org</em:id>
+    <em:version>1.0</em:version>
+    <em:type>2</em:type>
+    <em:bootstrap>true</em:bootstrap>
+    <em:unpack>false</em:unpack>
+    <em:multiprocessCompatible>true</em:multiprocessCompatible>
+
+    <!-- Firefox -->
+    <em:targetApplication>
+      <Description>
+        <em:id>{ec8030f7-c20a-464f-9b0e-13a3a9e97384}</em:id>
+        <em:minVersion>61.*</em:minVersion>
+        <em:maxVersion>63.*</em:maxVersion>
```

Comment:
With the limitation to Firefox 61-63, is this data collection still representative of the whole population?

Reply:
We intend to only serve this to Release channel (AFAIK), and 64 is going to disable Legacy extensions so we'll need to have this rewritten by then :)

Reply:
Can we enable this for <61 as well?

Reply:
Let's do 43+. A quick query showing we're still getting data from these versions: https://sql.telemetry.mozilla.org/queries/58776/source

Reply:
Hm, I *think* we only support system add-on updates back to 44 or so, I'll confirm and set it as far back as we can.

Reply:
That's fine if that's as far back as we can go. If this adds significant complexity I can restrict further too.

Reply:
Should be fine... doing QA for builds that far back can be a problem. I am pretty sure I am not using any code that is modern enough to be a problem (there have been some changes in the internal APIs in the meantime). The worst thing that can happen is that the extension throws an exception and does not work. I'll grab an old build just to see though.

One note though - since legacy extensions are going away in 64 we're going to need to rewrite this to be WebExtension-based, which is going to be a problem for going back more than a few releases. If we want to continue shipping to these older builds in the future then we'll need to maintain two extensions (legacy and WE) which I'd like to avoid if possible :)